docs/how-to-write-a-resolver.md

```diff
@@ -220,6 +220,32 @@ func (r *resolver) ValidateParams(ctx context.Context, params map[string]string)
 You'll also need to add the `"errors"` package to your list of imports at
 the top of the file.
 
+## The `ValidateUrl` method
```

Comment:
Do we need to mark a stability level here?

Reply:
I don't think that might help because the stability level is for the use of concise-resolver syntax. If that is enabled, the url will be non-empty and in-turn, this function will be called. The function itself has no stability level. But I can make a note of it anyway.

Reply:
bleh we're being inconsistent with API compatibility for resolver interface 😢 

We say that resolver interface is beta and [in-scope for API compatibility](https://github.com/tektoncd/pipeline/blob/main/api_compatibility_policy.md#backwards-incompatible-changes), but we also say [resolver interface is best-effort compatibility elsewhere in the doc](https://github.com/tektoncd/pipeline/blob/main/api_compatibility_policy.md#resolver-interface).

I'd default to beta == in scope for API compatibility. If you're unsure about the stability of the interface / might want to make additional changes I'd start this off as alpha and bump to beta once you have confidence this won't need additional changes.

Reply:
Fixed!